apps/docs/components/Navigation/NavigationMenu/HomeMenu.tsx

```diff
@@ -129,7 +129,7 @@ const NavigationMenuHome = () => {
       <ul className="relative w-full flex flex-col gap-4">
         {home.map((section, sectionIndex) => {
           return (
-            <>
+            <div key={`section-container-${sectionIndex}-border`}>
```

Comment:
nit: if let's say `<></>` is required, we can also do `<Fragment key=""></Fragment>` too

just fyi - nothing wrong with this :)

Reply:
ummm yes! That's true. I think that will fix another key issue actually. Thanks @joshenlim !